respy/pre_processing/specification_helpers.py

```diff
@@ -1,10 +1,12 @@
+import itertools
+
 import numpy as np
 import pandas as pd
 
 from respy.config import ROOT_DIR
 
 
-def csv_template(n_types, n_type_covariates, initialize_coeffs=True):
+def csv_template(n_types, n_type_covariates, observables, initialize_coeffs=True):
```

Comment:
Do not add observables here. Instead, use `point_constr` as in the case with lagged choices further below.